src/_cffi_src/openssl/ssl.py

```diff
@@ -733,22 +680,6 @@
 static const long TLS_ST_OK = 0;
 #endif
 
-/* LibreSSL 2.9.1 added only the DTLS_*_method functions */
-#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_102 && !CRYPTOGRAPHY_LIBRESSL_291_OR_GREATER
```

Comment:
Since we still support LibreSSL < 2.9.1 we need some logic here still right?

Reply:
Uhh, probably. I'm going to let the CI fail first though :D